flood_adapt/object_model/scenario.py

```diff
@@ -82,3 +100,28 @@ def __eq__(self, other):
         test2 = self.attrs.projection == other.attrs.projection
         test3 = self.attrs.strategy == other.attrs.strategy
         return test1 & test2 & test3
+
+    @staticmethod
+    def initiate_root_logger(filename):
```

Comment:
In the future, it would be nice to create a similar backend logger to capture stuff like this. But for now, this is fine